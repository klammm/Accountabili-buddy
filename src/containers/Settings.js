import React from 'react';
import { View, Text, StyleSheet } from 'react-native';

const styles = StyleSheet.create({
  container: {
    flex: 1,
    alignItems: 'center',
    justifyContent: 'center',
    backgroundColor: '#66A6FD',
  },
});

const Settings = () => {
  return (
    <View style={styles.container}>
      <Text>Me</Text>
    </View>
  );
};

export default Settings;
