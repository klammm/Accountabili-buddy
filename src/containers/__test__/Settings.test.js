import 'react-native';
import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import { Settings } from '../Settings';

describe('tests the Settings component', () => {
  it('renders Settings component', () => {
  const component = shallow(
    <Settings
      login={{
        email: '',
        password: '',
        error: '',
        loading: false,
        isLoggedIn: false
      }}
      logoutUser={jest.fn}
      onLogoutAttempt={jest.fn}
    />
  );
    expect(toJson(component)).toMatchSnapshot();
  });
});
