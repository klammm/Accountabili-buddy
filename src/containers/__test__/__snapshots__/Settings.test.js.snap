// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests the Settings component renders Settings component 1`] = `
<Card>
  <CardSection>
    <Button
      whenPressed={[Function]}
    >
      Log out
    </Button>
  </CardSection>
</Card>
`;
